eng/CodeAnalysis.ruleset

```diff
@@ -194,10 +194,10 @@
         <Rule Id="SA1630"  Action="None" />          <!-- Documentation text should contain whitespace -->
         <Rule Id="SA1631"  Action="None" />          <!-- Documentation should meet character percentage -->
         <Rule Id="SA1632"  Action="None" />          <!-- Documentation text should meet minimum character length -->
-        <Rule Id="SA1633"  Action="None" />          <!-- File should have header -->
-        <Rule Id="SA1634"  Action="None" />          <!-- File header should show copyright -->
-        <Rule Id="SA1635"  Action="None" />          <!-- File header should have copyright text -->
-        <Rule Id="SA1636"  Action="None" />          <!-- File header copyright text should match -->
+        <Rule Id="SA1633"  Action="Warning" />       <!-- File should have header -->
```

Comment:
Does this just enforce that one exists or does it enforce the value? If it enforces the value how does it know which one to require?

Reply:
I've been working with @chidozieononiwu with the files I used for the setup engine that enforced this. It's done through StyleCop and you can enforce non-XML headers. This change would enforce it, but the corresponding StyleCop.json changes (and how we put those as `AdditionalItems` items in each project) needs to be done as well.

Reply:
@chidozieononiwu was right, stylecop.json was ignored by .gitignore, it contains the required header value.